code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -91,6 +90,12 @@
 		if(3)
 			adjustBruteLoss(30)
 
+/mob/living/simple_animal/hostile/guardian/gib()
+	if(summoner)
+		src.summoner << "<span class='danger'><B>Your [src.name] was blown up!</span></B>"
```

Comment:
>`[src.name]`

The first thing wrong with this is that prefixing local variables with src is dumb if there isn't potential scope confusion
The second thing wrong is the fact that you're embedding the name at all; you just need to embed src and it does that for you
Make it `Your [src]`